reana_workflow_controller/k8s.py

```diff
@@ -137,12 +152,18 @@ def _build_deployment(self, metadata):
             enable_service_links=False,
             automount_service_account_token=False,
         )
+        labels = {
+            "app": self.deployment_name,
+            "reana_workflow_mode": "session",
+            "reana-run-session-workflow-uuid": str(self.workflow_id),
+            "reana-run-session-owner-id": str(self.owner_id),
```

Comment:
What do you think about standardizing labels name? I think we should decide whether we want to use `_` or `-` and whether we want to use the prefix `reana.io/...` (as we do for example for the `reana.io/system` label). This can be done in another PR.

Reply:
Created an issue for this: https://github.com/reanahub/reana-workflow-controller/issues/503